src/client/datascience/context/activeEditorContext.ts

```diff
@@ -112,7 +112,7 @@ export class ActiveEditorContextService implements IExtensionSingleActivationSer
         // This is temporary, and once we ship native editor this needs to be removed.
         setSharedProperty('ds_notebookeditor', e?.type);
         this.nativeContext.set(!!e).ignoreErrors();
-        this.isNotebookTrusted.set(e?.model?.isTrusted === true).ignoreErrors();
+        this.isNotebookTrusted.set(e?.model?.isTrusted !== false).ignoreErrors();
```

Comment:
```suggestion
        this.isNotebookTrusted.set(e?.model?.isTrusted).ignoreErrors();
```

Isn't this the same?

Reply:
Else the context variable gets set to `true`, which would be incorrect based on your comments.
I.e. 
* trusted = true (then yes)
* trusted = false (then no)
* trusted = undefined (then not used).

Shouldn't the same logic apply to this context variable?

Reply:
If trusted=undefined we want to show all commands for editing notebooks, so the context variable should be set to true. But I agree with Rich and I'm reverting these changes, will check in new code shortly